src/components/Delegates/DelegateCard/DelegateActions.tsx

```diff
@@ -36,6 +37,36 @@ export function DelegateActions({
     delegate.address.toLowerCase() as `0x${string}`
   );
 
+  const delegationButton = () => {
+    // TODO: Replace namespace lookup with governor-level switch in the tenant
+    switch (namespace) {
+      case TENANT_NAMESPACES.SCROLL:
```

Comment:
It would be great if at some point we could base these switches off of the type of governor, rather than the namespace. Is that possible or does that not make sense given the context here?

Reply:
(not a thing for this PR, just in general, I think if possible it's better to switch on governor type since its more general than tenant)

Reply:
IMHO, we should be doing it now, in this PR.  

There is a couple other spots where it's `namespace == SCROLL` added in this PR, but really it's GOV_TYPE == 'AGORA_V1'

Reply:
@jefag I didn't want to overload this PR with a bunch of tenant work but yes, it makes sense to introduce a governor-level switch.